components/capi/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 // This file is part of ICU4X. For terms of use, please see the file
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+#![allow(clippy::upper_case_acronyms)]
```

Comment:
You need an empty line after the copyright header

Reply:
fixed!